redaxo/src/addons/backup/lib/backup.php

```diff
@@ -137,7 +137,7 @@ public static function importDb($filename)
             try {
                 $sql->setQuery($line['query']);
             } catch (rex_sql_exception $e) {
-                $error .= "\n" . rex_escape($e->getMessage());
+                $error .= "\n" . nl2br(rex_escape($e->getMessage()));
```

Comment:
Ich meinte eigentlich das \n in dieser zeile :)

Reply:
Achso :D